CODE_OF_CONDUCT.md

```diff
@@ -52,23 +63,10 @@ address, posting via an official social media account, or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
-## Enforcement
```

Comment:
Where has this section gone?

Reply:
@StanczakDominik Some parts of it were already there in the last item of the pledge list above. Should I bring it back?